dashboard/app/controllers/api_controller.rb

```diff
@@ -425,6 +425,10 @@ def section_text_responses
 
   # GET /dashboardapi/sign_cookies
   def sign_cookies
+    # length of time the browser can privately cache this request for cookies
+    expires_in 1.hour
```

Comment:
Do we need to explicitly set `max-age` and `s-maxage` directives in the `Cache-Control` response header [the same way the script levels controller manages caching](https://github.com/code-dot-org/code-dot-org/blob/ec99e9665ded67a81a2eeabb90998e6375cf91b4/dashboard/app/controllers/script_levels_controller.rb#L219-L227).

Reply:
I take that back.  This is a private cache directive and should not be cached by intermediaries like CloudFront and Varnish.